sormas-ui/src/test/resources/META-INF/persistence.xml

```diff
@@ -81,7 +81,7 @@
 		<class>de.symeda.sormas.backend.vaccination.LastVaccineType</class>
 		<class>de.symeda.sormas.backend.vaccination.LastVaccinationDate</class>
 		<class>de.symeda.sormas.backend.vaccination.FirstVaccinationDate</class>
-		<class>de.symeda.sormas.backend.labmessage.TestReport</class>
+		<class>de.symeda.sormas.backend.externalmessage.TestReport</class>
```

Comment:
This comment has nothing to do with this line, but I've experienced that general comments are a rather hight risk of getting overlooked.

When I the code for LAB_MESSAGE, I get some results that should not be there, for example in the SystemEventType,java.

Also, adaptions the the SORMAS-DEMIS-adapter are necessary before merging this ([#61](https://github.com/hzi-braunschweig/sormas-demis-adapter/issues/61))